app/renderer/js/utils/logger-util.js

```diff
@@ -88,6 +95,20 @@ class Logger {
 	reportSentry(err) {
 		captureException(err);
 	}
+
+	trimLog(file) {
+		fs.readFile(file, 'utf8', (err, data) => {
+			if (err) {
+				throw err;
+			}
+			// Keep bottom MAX_LOG_FILE_LINES of each log instance
+			const MAX_LOG_FILE_LINES = 500;
+			const logs = data.split(os.EOL);
+			const trimmedLogs = logs.slice(Math.max(logs.length - MAX_LOG_FILE_LINES, 0));
```

Comment:
For some reason `logs.length` is returning one value more than the number of lines (not sure why) , so maybe add a -1 to `logs.length - MAX_LOG_FILE_LINES - 1` along with a comment.
Unless you can find out why thats occurring.

Reply:
I think that is because split by EOL is taking a last empty line as the last element in the array.

Reply:
Made the update.